script.js

```diff
@@ -0,0 +1,206 @@
+// ############################## init ##############################
+// localStorage에서 todo elements 가져오기
+const unparsedTODOS = localStorage.getItem('todos');
+const TODOS = unparsedTODOS
+  ? JSON.parse(unparsedTODOS).map((todo, idx) => ({
+      ...todo,
+      idx,
+      fromDate: new Date(todo.fromDate),
+      toDate: new Date(todo.toDate),
+    }))
+  : [];
+
+let nextIdx = TODOS.length; // idx값을 중복되지 않게 설정하도록 초기값 지정
+const ul = document.querySelector('.todoList'); // child node를 추가/제거 하기 위해 변수에 저장
+
+// in memory에 저장된 todo list를 view에 display
+for (const todo of TODOS) {
+  addNewTodoLi(todo);
+}
```

Comment:
이미 알고 계시겠지만 이부분에 고차함수를 써보는 것도 괜찮을 것 같아요!
```suggestion
TODOS.forEach(todo => addNewTodoLi(todo));
```